app/goal_routes.py

```diff
@@ -0,0 +1,93 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, abort, jsonify
+from app.routes_helpers import validate_model
+from app.models.goal import Goal
+
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+
+    if not request_body.get("title"):
+        abort(make_response({"details": "Invalid data"}, 400))
```

Comment:
How could we make this error message more descriptive for the user to understand what is missing? Maybe we can say that `title is missing` or something!